CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ## [Unreleased]
 
+### Migration steps
+
+- **Before upgrading to this release of Cumulus**, which enables integration with RDS and a PostgreSQL database for archiving Cumulus data, you will need to run a data migration to copy some of your data from DynamoDB to RDS. See the [upgrade documentation on running data migrations for this release](https://nasa.github.io/cumulus/docs/upgrade-notes/upgrade_tf_version_0.13.6).
```

Comment:
```suggestion
- **Before upgrading to this release of Cumulus**, which enables integration with a PostgreSQL database for archiving Cumulus data, you will need to run a data migration to copy some of your data from DynamoDB to RDS. See the [upgrade documentation on running data migrations for this release](https://nasa.github.io/cumulus/docs/upgrade-notes/upgrade_tf_version_0.13.6).
```

Consider removing RDS as it's not technically a prerequisite.

Reply:
I wrote this, so it's my bad. But I think we should call out in the CHANGELOG the separate steps of creating a PostgreSQL database and running the data migrations. Like so?

```suggestion
- **Before upgrading to this release of Cumulus**, which enables integration with a PostgreSQL database for archiving Cumulus data, you will need to:
  - Create a PostgreSQL database. [See the docs on how to deploy a database as part of a new RDS cluster](https://nasa.github.io/cumulus/docs/deployment/postgres-database-deployment.md).
  - Run a data migration to copy some of your data from DynamoDB to PostgreSQL. See the [upgrade documentation on running data migrations for this release](https://nasa.github.io/cumulus/docs/upgrade-notes/data_migrations_rds). There will also be some data migration steps post-deployment that are covered in the document.
```

Then we can remove the part about "create RDS cluster" from the migration steps document